src/ruleset.xml

```diff
@@ -4,7 +4,19 @@
 
     <rule ref="PSR2" />
 
+    <!-- Checks that the opening PHP tag is the first content in a file -->
     <rule ref="Generic.PHP.CharacterBeforePHPOpeningTag" />
+    <!-- Checks format of doc blocks -->
+    <rule ref="Generic.Commenting.DocComment" />
+    <!-- forbids TODO commemts -->
+    <rule ref="Generic.Commenting.Todo"/>
```

Comment:
I'm torn on this one. Todo's can be helpful future pointers... but we don't want to accidentally leave them in either

Reply:
yeah. So Will had the idea that TODOs should just become an issue and not live in the code, I think that'd be ideal, since we don't usually go back to cleanup TODOs otherwise. 

Maybe a decent approach would be that instead of a TODO we have an issue link with some other flag like "ISSUE" that we can check for format with a custom linter (there might be something that does this already) ? another idea is to forbid TODOs but only if they are not followed by an issue link. Not sure. Throwing ideas around :)